msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages_eu.properties

```diff
@@ -914,3 +914,8 @@ pvt_read_receipt_text=Jaso jakinarazpen bat hartzaileak mezua irekitzen duenean
 
 pvt_read_receipt_email=`{0}`-ek `{1}` mezua irakurri du {2}an
 pvt_read_receipt_email_subject={0} irakurri da
+
+#S2U-28
+pvt_tags_header=Etiquetas
+pvt_tags_save=Guardar Etiquetas
+pvt_enter_search_tags=Introduzca el texto de b\u00fasqueda.
```

Comment:
@PabloSanRoman would you mind providing translations for EU?

Reply:
Hi @bgarciaentornos 
pvt_tags_header=Etiketak
pvt_tags_save=Gorde etiketak
pvt_enter_search_tags=Sartu bilatzeko testua

Reply:
This is included in https://github.com/sakaiproject/sakai/pull/11627